src/scripts/wordle_guesser.js

```diff
@@ -37,7 +40,8 @@ class WordleGame {
   }
 
   isWordValid(word) {
-    return word.split('').every((char, i) => this.possibleLetters[i].includes(char));
+    return word.split('').every((char, i) => this.possibleLetters[i].includes(char))
+      && this.confirmedLetters.every((char) => word.includes(char));
```

Comment:
love your addition! should solve the issue you noticed. it took me a hot second to parse these both tbh, was thinking of pulling these out into named variables but honestly I think the names would be more complicated than the evaluations :joy:

might be a dumb q, but do you have to split the `word` string again before you can use `.includes`?

Reply:
Added named variables. They are bad names, but hopefully descriptive?